From e619fe74662c64e52791e673d7c1b6c0cb590bc2 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Mon, 12 Apr 2021 13:07:51 -0500
Subject: [PATCH 120/120] Revert "HSD #16013024105: fpga: stratix10-soc: stop
 kernel thread"

commit e05fb79f6c9c293dd3e52a2423ac29add29af434 from
https://github.com/altera-opensource/linux-socfpga.git

This reverts commit c7bfe97dd00781aee819c4cb0a75bed549f9037f.

Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/fpga/stratix10-soc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/fpga/stratix10-soc.c b/drivers/fpga/stratix10-soc.c
index 6ba59841fbf7..9ab7afd28a64 100644
--- a/drivers/fpga/stratix10-soc.c
+++ b/drivers/fpga/stratix10-soc.c
@@ -486,12 +486,10 @@ static int s10_probe(struct platform_device *pdev)
 			       NULL, 0, s10_fw_version_callback);
 	if (ret) {
 		dev_err(dev, "couldn't get firmware version\n");
-		stratix10_svc_done(priv->chan);
 		fpga_mgr_free(mgr);
 		goto probe_err;
 	}
 
-	stratix10_svc_done(priv->chan);
 	platform_set_drvdata(pdev, mgr);
 	return ret;
 
-- 
2.31.1

